config/releases.exs

```diff
@@ -27,7 +27,11 @@ config :magpie, Magpie.Repo,
   adapter: Ecto.Adapters.Postgres,
   url: System.fetch_env!("DATABASE_URL"),
   pool_size: String.to_integer(System.get_env("POOL_SIZE", "2")),
-  ssl: true,
+  ssl: (if System.get_env("DATABASE_SSL", "true") == "true" do
```

Comment:
The settings in `releases.exs` will only be relevant if the app is built via the native release mechanism with `mix release` (which is included as one of the steps in the Heroku buildpacks we're using). If we add the `DATABASE_SSL` environment variable, it seems to me that we'll also have to remember setting it in all the Heroku instances we deploy to.

Though I'm still not sure I got why this fixes running the app in Docker. (I'm sure I'm just out of touch with the way it's intended to be run.) Could you edify me on this? Thanks :)

Reply:
> Though I'm still not sure I got why this fixes running the app in Docker.

When running the container with docker-compose DATABASE_SSL=false is much easier to set up and should still be safe.

> If we add the DATABASE_SSL environment variable, it seems to me that we'll also have to remember setting it in all the Heroku instances we deploy to (and add this step to the README).

We could change the default from this to 'true' and only set it to false if the ENV var is false (which is the case in [the magpie-base CI config](https://github.com/magpie-ea/magpie-base/blob/develop/.github/workflows/integration-test.yml#L41))

Reply:
Yep, I think this would make it much easier 👍 . Otherwise the existing deployment would break.